src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
```

Comment:
Replace the hard-coded value `3` with a constant that explains what this value represents (e.g., `SHORT_TERM_RENTAL_DAYS`). Also, replace the hard-coded value `20` with a constant (e.g., `SHORT_TERM_DISCOUNT`). See checklist item 2.